lucene/core/src/java/org/apache/lucene/codecs/blocktree/Stats.java

```diff
@@ -75,6 +75,17 @@
   /** Total number of bytes used to store term suffixes. */
   public long totalBlockSuffixBytes;
 
+  /**
+   * Number of times each compression method has been used.
+   * 0 = uncompressed
+   * 1 = lowercase_ascii
+   * 2 = LZ4
+   */
+  public final long[] compressionAlgorithms = new long[3];
```

Comment:
Cool that you track this in BlockTree stats!  Did you post the stats somewhere?  Edit: ahh, I see the [cool stats here](https://issues.apache.org/jira/browse/LUCENE-4702?focusedCommentId=17003640&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17003640), thanks.